requirements.txt

```diff
@@ -4,9 +4,9 @@
 CherryPy
 requests>=2.7.0
 importlib
-pbr
```

Comment:
I did not yet removed in setup.py .. because some features in setup.py still require `pbr` but it is not anymore in the requirements that are used by the packaging.